demo/src/app/examples/advanced/tabs-bootstrap/tab-type.component.ts

```diff
@@ -72,8 +71,8 @@ export class TabTypeComponent extends FieldType implements OnInit {
   }
 
   isValid(index): boolean {
-    let tab = this.steps[index];
-    for (let subfield of tab.fields) {
+    let tab = this.field.fieldGroup[index];
```

Comment:
```ts
return this.field.fieldGroup[index].formControl.valid;
```

Reply:
Not work, because the formcontrol contains all the fields and not only the fields of the tab.

Reply:
ok no worry, I'll check it later!